pam/integration-tests/testdata/TestGdmModuleWithCWrapper

```diff
@@ -0,0 +1 @@
+TestGdmModule
```

Comment:
hum, sorry, what’s the purpose of this file?

Reply:
It's a symlink to the `TestGdmModule` where there are the some golden files.

Since we just repeat the same tests, having it as a link instead of a copy was ensuring that they should work in the same piece of data.

Reply:
Oh, I thought that github would do a better job showing up a symlink in the diff, I see on the side that indeed, it’s a symlink.